packages/network/src/v2/schemas/decodeStaticField.spec.ts

```diff
@@ -0,0 +1,63 @@
+import { SchemaType } from "@latticexyz/schema-type";
+import { hexToArray } from "@latticexyz/utils";
+import { decodeStaticField } from "./decodeStaticField";
+
+describe("decodeStaticField", () => {
```

Comment:
This isn't a full suite but I figured a few tests were better than nothing and helped me to get the decoding fixed